screenpy/actions/make_note.py

```diff
@@ -63,6 +63,9 @@ def perform_as(self, the_actor: Actor) -> None:
             # must be a value instead of a question!
             value = self.question
 
+        if hasattr(self.question, "caught_exception"):
+            aside(f"{self.question.caught_exception}")
```

Comment:
I think we probably want to add some context to this log, otherwise it will look like there's an uncaught exception. Maybe something like `"Making note of {self.question} caught this exception:\n{self.question.caught_exception}"`?

Reply:
@perrygoy You were right.  This needed some context.  I thought there would be enough context from the previous log line from `MakeNote` but I didn't realize it doesn't mention what the question was.  Committed new changes.